pyproject.toml

```diff
@@ -64,13 +64,13 @@ dependencies = [
     "jinja2>=3.1.3",
     "tenacity>=8.2",
     "puremagic>=1.20",
-    "pymupdf>=1.24.1",
     "ftfy>=6.2.0",
     "pydantic>=2.7.4",
     "pyarrow>=19",
     "pixeltable-pgserver==0.4.0",
     "tzlocal>=5.0",
     "aiohttp>=3.9",
+    "pypdfium2>=5.1.0",
```

Comment:
Let's do >=5.0.0. Even that is very recent.

Reply:
Done. But does this change which actual version is used? I feel like it doesn't as uv lock did not change.